### PR TITLE
Make all entity links go through the `link` path.

### PR DESCRIPTION
**Pull request by mcbloch**

### CODE REVIEW

**Comment by mcbloch** [comment]
Could this also be approved, otherwise the backend is not going to work ^^ @maartenvn

**Review by maartenvn — APPROVED** [review]
I was waiting until the changes in the backend were merged (the mocking server will no longer work without the changes to the OpenAPI spec)

The rest seems ok

**Comment by Jan-PieterBaert** [comment]
since @maartenvn said that this will no longer work without changes to the OpenAPI spec, perhaps we should update the spec?

**Pull request merged**
